spec/sidekiq/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -15,6 +15,8 @@
     Flipper.disable(:disability_526_classifier_new_claims)
     Flipper.disable(:disability_compensation_lighthouse_claims_service_provider)
     Flipper.disable(:disability_526_classifier_multi_contention)
+    Flipper.disable(:disability_compensation_production_tester)
+    Flipper.disable(:disability_compensation_fail_submission)
```

Comment:
I'm guessing this is the only place where backup submission is tested?

Reply:
Correct. The backup submission process has its own internal tests, but since we're actively bypassing the process there's no need to modify them.